k8s/prod/common/probate/business-service.yaml

```diff
@@ -0,0 +1,37 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: probate-business-service
+  namespace: probate
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: probate-business-service
+  rollback:
+    enable: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: probate-business-service
+    version: 0.0.3
+  values:
+    java:
+      replicas: 2
```

Comment:
Please add the limits and requests from my other PR:
https://github.com/hmcts/cnp-flux-config/pull/569

Reply:
done